internal/runtime/runtime.go

```diff
@@ -27,15 +28,19 @@ import (
 	"github.com/ugorji/go/codec"
 )
 
+// AppFunction type alias
+type AppFunction = func(edgexcontext *appcontext.Context, params ...interface{}) (bool, interface{})
```

Comment:
Why is this type redefined here? It's already in appsdk/sdk.go. Declarations are identical.

Reply:
I moved it to context which means it no longer needs to be re-declared everywhere and can now be used wherever.